compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -721,16 +721,6 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
   CHECK_OR_FALSE(is_instance_mean_v2(mean_of_ifm));
   CHECK_OR_FALSE(ifm == mean_of_ifm->input());
 
-  // If const_as_beta has shape of '1 x chennel x (1 or input last dimension)'
-  uint32_t input_channel = ifm_circle->dim(1).value();
-  uint32_t input_last_dim = ifm_circle->dim(2).value();
-  const_as_beta = dynamic_cast<luci::CircleConst *>(sub->x());
-  CHECK_OR_FALSE(const_as_beta);
-  CHECK_OR_FALSE(const_as_beta->rank() == 3);
-  CHECK_OR_FALSE(
-    const_as_beta->dim(0).value() == 1 && const_as_beta->dim(1).value() == input_channel &&
-    (const_as_beta->dim(2).value() == 1 || const_as_beta->dim(2).value() == input_last_dim));
```

Comment:
We may need to check the shape and the value of sub->x() (`const_as_zero`).

Reply:
So we need to check all the values should be 0.0f too ?